src/show/pages/private/Admin/index.jsx

```diff
@@ -0,0 +1,718 @@
+            <Button
+              label='+ Event'
+              variant='primary'
+              onClick={() => setCreateEventModal(true)}
+            />
+          </div>
+          <div className='h-full w-full'>
+            <div className='w-full h-full grid grid-cols-5 gap-1 overflow-y-auto'>
+              {constructors.length > 0 ? (
+                constructors?.map(constructor => (
+                  <ConstructorCardAdmin
+                    key={constructor.id}
+                    constructor={constructor}
+                    logo_url={constructor.logo_url}
+                    constructors={constructors}
+                    setConstructors={setConstructors}
+                  />
+                ))
+              ) : (
+                <div className='flex w-full h-full t justify-center text-secondary'>
+                  No Constructors yet
+                </div>
+              )}
+              {drivers.length > 0 ? (
+                drivers?.map(driver => (
+                  <DriverCardAdmin
+                    key={driver.id}
+                    driver={driver}
+                    drivers={drivers}
+                    setDrivers={setDrivers}
+                  />
+                ))
+              ) : (
+                <div className='flex w-full h-full justify-center text-secondary'>
+                  No Drivers yet
+                </div>
+              )}
+              {events.length > 0 ? (
+                events?.map(event => (
+                  <EventCardAdmin
+                    key={event.id}
+                    event={event}
+                    events={events}
+                    setEvents={setEvents}
+                  />
+                ))
+              ) : (
+                <div className='flex w-full h-full justify-center text-secondary'>
+                  No Events yet
+                </div>
+              )}
+            </div>
+          </div>
+        </div>
+        <Modal setVisible={setCreateDriverModal} visible={createDriverModal}>
+          <div className='min-h-full w-full flex items-center justify-center py-3'>
+            <div className='max-w-md w-full'>
+              <div className='flex flex-col w-full justify-center items-center'>
+                <h2 className='mb-10 text-center text-3xl font-extrabold text-charade'>
+                  Create a new Driver
+                </h2>
+              </div>
+              <div className='flex flex-col space-y-6'>
+                <div className='flex flex-col w-full space-y-5'>
+                  <label className='flex w-full items-start'>
+                    <div className='flex w-1/5'>First Name:</div>
+                    <div className='w-4/5'>
+                      <TextField
+                        required
+                        type='text'
+                        placeholder='First Name'
+                        className='rounded-md w-full h-8 bg-white-600 text-center outline-secondary outline-offset-4'
```

Comment:
Why set this class name here 26 times instead of doing it one time in the text field component.

Reply:
done